src/rendering/mod.rs

```diff
@@ -5,7 +5,8 @@ use egui::{
 };
 use smithay::{
     backend::renderer::gles2::{ffi, Gles2Error, Gles2Frame, Gles2Renderer},
-    utils::{Physical, Point, Rectangle, Size},
+    nix::sys::socket::sockopt::ReceiveTimeout,
```

Comment:
?

Reply:
vscode code completion must have add that when I typed something 😩